onadata/apps/api/tests/viewsets/test_organization_profile_viewset.py

```diff
@@ -269,6 +271,28 @@ def test_add_members_to_org(self):
         self.assertEqual(response.status_code, 201)
         self.assertEqual(set(response.data), set([u'denoinc', u'aboy']))
 
+    def test_add_members_to_org_user_org_account(self):
+        self._org_create()
+        view = OrganizationProfileViewSet.as_view({
+            'post': 'members'
+        })
+
+        username = 'second_inc'
+
+        # Create second org
+        org_data = {'org': username}
+        self._org_create(org_data=org_data)
+
+        data = {'username': username}
+        request = self.factory.post(
+            '/', data=json.dumps(data),
+            content_type="application/json", **self.extra)
+
+        response = view(request, user='denoinc')
+        self.assertEqual(response.status_code, 400)
+        self.assertEqual(response.data,
+                         u"Cannot add org account `second_inc` as member.")
```

Comment:
We probably should have this functionality and everything in members handled by serializers. I would not expect this response from the API, it usually is a JSON response and not a string.